src/components/HorizontalCard/index.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react'
+
+import styles from './HorizontalCard.module.scss'
+
+import classNames from 'classnames'
+
+export interface HorizontalCardProps
+  extends React.HTMLAttributes<HTMLDivElement> {
+  gameImage?: React.JSX.Element
+  imageUrl?: string
+  children?: React.ReactNode
+  title: string
+  orderNumber?: string
+  onCardClick: React.MouseEventHandler<HTMLDivElement>
+  tone?: 'brand' | 'neutral'
+  size?: 'large' | 'small'
+}
+
+const HorizontalCard: React.FC<HorizontalCardProps> = ({
+  gameImage,
+  title,
+  children,
+  orderNumber,
+  onCardClick,
+  tone = 'neutral',
+  size = 'large'
+}) => {
+  const image = gameImage || <img src="/fallback_card.jpg" alt="Game Image" />
```

Comment:
I think this may break when you pull it into the store unless you import the FallbackImage like we do in the `GameCard`:

```suggestion
import styles from './HorizontalCard.module.scss'

import classNames from 'classnames'
import FallbackImage from '@/assets/fallback_card.jpg?url'

export interface HorizontalCardProps
  extends React.HTMLAttributes<HTMLDivElement> {
  gameImage?: React.JSX.Element
  imageUrl?: string
  children?: React.ReactNode
  title: string
  orderNumber?: string
  onCardClick: React.MouseEventHandler<HTMLDivElement>
  tone?: 'brand' | 'neutral'
  size?: 'large' | 'small'
}

const HorizontalCard: React.FC<HorizontalCardProps> = ({
  gameImage,
  title,
  children,
  orderNumber,
  onCardClick,
  tone = 'neutral',
  size = 'large'
}) => {
  const image = gameImage || <img src={FallbackImage} alt="Game Image" />
```